### PR TITLE
Link to ?read.delim in ?fread to give a closer analogue of expected behavior

### PR DESCRIPTION
**Pull request by MLopez-Ibanez**
See #5634

### CODE REVIEW

File: man/fread.Rd [hunk 1]
```diff
@@ -2,7 +2,7 @@
 \alias{fread}
 \title{ Fast and friendly file finagler }
 \description{
-   Similar to \code{read.table} but faster and more convenient. All controls such as \code{sep}, \code{colClasses} and \code{nrows} are automatically detected.
+   Similar to \code{\link[utils:read.delim]{read.delim()}} but faster and more convenient. All controls such as \code{sep}, \code{colClasses} and \code{nrows} are automatically detected.
```
Comment (MichaelChirico) [inline man/fread.Rd:5]: Hmm, I actually don't think this is right.

`read.csv()` and `read.delim()` are thin wrappers around `read.table()`. The former sets `sep=","`, the latter `sep="\t"`.

However, `fread()` uses `sep="auto"` by default, meaning it's more generic than either `read.csv()` or `read.delim()`. Thus I think `read.table()` is the best fit (even though `fread()`'s auto-detection is not available in `read.table()`).

Partially, this is splitting hairs -- `?read.table`, `?read.csv`, and `?read.delim` are all the same page.

```suggestion
   Similar to \code{\link[utils:read.table]{read.table()}} but faster and more convenient. All controls such as \code{sep}, \code{colClasses} and \code{nrows} are automatically detected.
```

Comment (MichaelChirico) [reply man/fread.Rd:5]: Sorry, I've just read the referenced discussion (FFR: use `#<issue>` in the PR body to make it easier on reviewers -- numbers in the PR title aren't auto-linked).

Maybe the compromise here is "Similar to [read.csv()] or [read.delim()]", that way we convey that it's for delimited files but not implying the full generality of `read.table()`, WDYT?

Comment (MLopez-Ibanez) [reply man/fread.Rd:5]: Sounds OK. Thanks.

**Comment by MichaelChirico** [comment]
Thanks!

**Pull request merged**